configs/webpack/base.coffee

```diff
@@ -26,19 +26,19 @@ STATICS =
   image: { test: /\.(png|jpg|svg|gif)/,    use: [ LOADERS.file ] }
   font:  { test: /\.(woff|woff2|eot|ttf)/, use: [ LOADERS.url  ] }
 
+extractText = ExtractTextPlugin.extract({
```

Comment:
heh, nope.  That was leftover from an attempt to share the plugin instance between the different loaders, which  didn't work